base/test/mp4writersink_tests.cpp

```diff
@@ -29,7 +29,7 @@ void writeH264(bool readLoop, int sleepSeconds, std::string outFolderPath, int c
 	Logger::initLogger(loggerProps);
 
 	auto fileReaderProps = FileReaderModuleProps(inFolderPath, 0, -1);
-	fileReaderProps.fps = 100;
+	fileReaderProps.fps = 24;
```

Comment:
Add two tests

Test1 (without metadata)
1. Use a fixed data to write mp4 video using step interface 
2. Verify the size of the final video file with some tolerance (if reqd)

Test2 (with metadata - for every 6th frame)
1. Use a fixed data to write mp4 video using step interface 
2. Verify the size of the final video file with some tolerance (if reqd)


See if other tests are required. If not, clean them up.

Reply:
added